ci/run_ci.sh

```diff
@@ -253,6 +253,9 @@ case $1 in
       cd "$ROOT/java"
       mvn -T10 -B --no-transfer-progress spotless:check
       mvn -T10 -B --no-transfer-progress checkstyle:check
+      pushd "$ROOT/javascript"
```

Comment:
There is already a checker in ci/format.sh

Reply:
Yes, I will submit a commit remove this redundant check